rag_experiment_accelerator/doc_loader/documentIntelligenceLoader.py

```diff
@@ -88,13 +87,13 @@ def load_with_azure_document_intelligence(
     logger.debug(f"Loaded {len(documents)} documents using Azure Document Intelligence")
 
     text_splitter = RecursiveCharacterTextSplitter(
```

Comment:
Severity: Major - Using `index_config` for `chunk_size` and `chunk_overlap` centralizes configuration management, improving maintainability. Ensure that these values are validated to prevent runtime errors.